src/components/selection.ts

```diff
@@ -34,6 +34,34 @@ interface Document {
  * @typedef {SelectionUtils} SelectionUtils
  */
 export default class SelectionUtils {
+  /**
+   * Selection instances
+   *
+   * @todo Check if this is still relevant
+   */
+  public instance: Selection = null;
```

Comment:
why the instance went public?

Reply:
It was public... Just moved it to the top cause linter was angry